facebook.js

```diff
@@ -1,78 +1,148 @@
+var stad = chrome.i18n.getMessage("stand");
+var per = chrome.i18n.getMessage("per");
```

Comment:
I think chrome.i189n would work in most of the cases and even if we used something like this:
```
var locale = (Array.from(document.body.classList).find(cls => cls.match(/^Locale_/)));
if(locale == "Locale_en_ES"){
  
}
```
Then, we would still need to create string variables for each keys?